Source/SPStringAdditions.m

```diff
@@ -382,6 +382,46 @@ - (NSString *)stringByReplacingCharactersInSet:(NSCharacterSet *)set withString:
 	return newString;
 }
 
+// from Apple: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Strings/Articles/SearchingStrings.html#//apple_ref/doc/uid/20000149-SW7
+- (BOOL)hasPrefix:(NSString *)prefix caseInsensitive:(BOOL)caseInsensitive
+{
+	if (!caseInsensitive){
+        return [self hasPrefix:prefix];
+	}
+	
+    const NSStringCompareOptions options = NSAnchoredSearch|NSCaseInsensitiveSearch;
+	
+    NSRange prefixRange = [self rangeOfString:prefix options:options];
+    
+	return prefixRange.location == 0 && prefixRange.length > 0;
+}
+
+- (NSString *)trimSubstringFromStart:(NSString *)needle
```

Comment:
As I mentioned - this can be super nicely achieved in swift - can we have `StringExtension.swift` swift file with these helper methods? Do you want me to implement them and help ya?

Reply:
sure, show mw how much more simple it is... ;)

Reply:
You got it :) https://github.com/Sequel-Ace/Sequel-Ace/pull/264